docs/tutorial/arguments/optional.md

```diff
@@ -109,21 +109,27 @@ name: str
 
 Now, finally what we came for, an optional *CLI argument*.
 
-To make a *CLI argument* optional, use `typer.Argument()` and pass a different "default" as the first parameter to `typer.Argument()`, for example `None`:
+To make a *CLI argument* optional, use `typer.Argument()` and make sure to provide a "default" value, for example `"World"`:
 
 {* docs_src/arguments/optional/tutorial002_an.py hl[7] *}
 
 Now we have:
 
 ```Python
-name: Annotated[Optional[str], typer.Argument()] = None
+name: Annotated[str, typer.Argument()] = "World"
 ```
 
 Because we are using `typer.Argument()` **Typer** will know that this is a *CLI argument* (no matter if *required* or *optional*).
 
 /// tip
 
-By using `Optional` your editor will be able to know that the value *could* be `None`, and will be able to warn you if you do something assuming it is a `str` that would break if it was `None`.
+If you want the default value to be `None`, you have to additionally declare the parameter to be of type `Optional`, which will tell your editor that the value of this parameter can be `None`:
+
+```Python
+name: Annotated[Optional[str], typer.Argument()] = None
+```
+
+Your editor can then warn you if you do something assuming it is a `str` that would break if it was `None`.
```

Comment:
I'm not even sure we need all this additional explanation about `None` - it may still cause some confusion and we're probably already explaining `Optional` in various other places.

Reply:
Agreed! I'm gonna remove it. 🤓